english/yoga/mind/yoga_mind.md

```diff
@@ -0,0 +1 @@
+You close your eyes and experience a clarity of vision, supreme focus, a feeling of oneness with the universe... you are one with the mushroom!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.